test/new_relic/helper_test.rb

```diff
@@ -77,4 +77,28 @@ def test_run_command_sad_exception
       NewRelic::Helper.run_command('executable that existed at detection time but is not there now')
     end
   end
+
+  #
+  # rubygems_specs
+  # 
+  def test_rubygems_specs_returns_empty_array_without_bundler
+    stub(:defined?, nil, ['Bundler']) do
+      result = NewRelic::Helper.rubygems_specs
+
+      assert_instance_of Array, result
+      assert_empty Array, result
+    end
+  end
+
+  def test_rubygems_specs_works_with_all_specs_when_installed_specs_is_absent
+    Bundler.rubygems.stub(:respond_to?, nil) do
```

Comment:
I don't understand why, but Rubies 2.6 and below don't like the third argument for a stub in this scenario. However, they're fine with it in the test above. 🤷

Reply:
My guess is that the older Rubies are given an older Minitest. Wanting to use 3 arguments was the primary reason for the introduction of our `skip_unless_minitest5_or_above` helper.